lib/l10n/app_id.arb

```diff
@@ -289,36 +289,36 @@
   "friends": "Daftar Teman",
   "no_lyrics_available": "Maaf, tidak dapat menemukan lirik untuk lagu ini",
   "start_a_radio": "Putar Radio",
-  "how_to_start_radio": "Bagaimana Anda ingin memutar radio?",
-  "replace_queue_question": "Apakah Anda ingin mengganti antrean saat ini atau menambahkannya?",
+  "how_to_start_radio": "Bagaimana cara memutar radio?",
+  "replace_queue_question": "Apakah kamu ingin mengganti antrean saat ini atau menambahkannya?",
   "endless_playback": "Pemutaran Tanpa Akhir",
   "delete_playlist": "Hapus Daftar Putar",
-  "delete_playlist_confirmation": "Anda yakin ingin menghapus daftar putar ini?",
+  "delete_playlist_confirmation": "Kamu yakin ingin menghapus daftar putar ini?",
   "local_tracks": "Trek Lokal",
   "song_link": "Tautan Lagu",
   "skip_this_nonsense": "Lewati omong kosong ini",
   "freedom_of_music": "“Kebebasan Musik”",
-  "freedom_of_music_palm": "“Kebebasan Musik di telapak tangan Anda”",
+  "freedom_of_music_palm": "“Kebebasan Musik di telapak tanganmu",
   "get_started": "Mari kita mulai",
   "youtube_source_description": "Direkomendasikan dan berfungsi paling baik.",
   "piped_source_description": "Merasa bebas? Sama seperti YouTube tetapi banyak yang gratis.",
   "jiosaavn_source_description": "Terbaik untuk wilayah Asia Selatan.",
   "highest_quality": "Kualitas Terbaik: {quality}",
   "select_audio_source": "Pilih Sumber Suara",
   "endless_playback_description": "Tambahkan lagu baru secara otomatis\nke akhir antrean",
-  "choose_your_region": "Pilih wilayah Anda",
-  "choose_your_region_description": "Ini akan membantu Spotube menampilkan konten yang tepat\nuntuk lokasi Anda.",
-  "choose_your_language": "Pilih bahasa Anda",
+  "choose_your_region": "Pilih wilayah",
+  "choose_your_region_description": "Ini akan membantu Spotube menampilkan konten yang tepat\nuntuk lokasimu.",
```

Comment:
`"choose_your_region_description": "Ini akan membantu Spotube menampilkan konten yang tepat\nuntuk lokasimu.",`
Would be better 
`"choose_your_region_description": "Ini akan membantu Spotube menampilkan konten yang tepat\nsesuai lokasimu.",`